api/src/main/java/io/opencensus/metrics/MetricRegistry.java

```diff
@@ -159,6 +159,64 @@ public DerivedDoubleGauge addDerivedDoubleGauge(
   @ExperimentalApi
   public abstract DerivedDoubleGauge addDerivedDoubleGauge(String name, MetricOptions options);
 
+  /**
+   * Builds a new long cumulative to be added to the registry. This is more convenient form when you
+   * want to manually increase values as per your service requirements.
+   *
+   * @param name the name of the metric.
+   * @param options the options for the metric.
+   * @return a {@code LongCumulative}.
+   * @throws NullPointerException if {@code name} is null.
+   * @throws IllegalArgumentException if different metric with the same name already registered.
+   * @since 0.21
+   */
+  @ExperimentalApi
+  public abstract LongCumulative addLongCumulative(String name, MetricOptions options);
```

Comment:
Why the name `add..` - seems like it's only building something not adding it to anything.

Reply:
It's both building a new `*Cumulative` object and adding it to the `MetricRegistry` so that all metric readers can read it from the `MetricRegistry`.